tests/test_converter.py

```diff
@@ -1176,6 +1176,66 @@ def test_qasm_(self, recorder):
         assert recorder.queue[1].parameters == []
         assert recorder.queue[1].wires == Wires([2, 0])
 
+        for i in range(2, 6):
+            assert recorder.queue[i].name == "MidMeasureMP"
+            assert recorder.queue[i].wires == Wires([i - 2])
+
+    def test_qasm_measure(self):
+        """Tests that measurements specified as an argument are added to the converted circuit."""
+        qasm_string = 'include "qelib1.inc";' "qreg q[2];" "creg c[2];" "h q[0];" "cx q[0], q[1];"
```

Comment:
Reformatting this string will fix the formatting complaints in CI regarding this line. For details on what the formatting check is catching, you can see [the documentation here](https://peps.python.org/pep-3126/)